Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -82,20 +82,23 @@ namespace OpenInfraPlatform {
 					{
 
 					// (1/3) IfcBoundedSurface SUBTYPE of IfcSurface
-					if (surface.isOfType<typename IfcEntityTypesT::IfcBoundedSurface>()) {
-						return std::shared_ptr<carve::input::PolylineSetData> polylineData = convertIfcBoundedSurface(surface.as<typename IfcEntityTypesT::IfcBoundedSurface(),
+					if (surface.isOfType<typename IfcEntityTypesT::IfcBoundedSurface>()) 
+					{
+						return convertIfcBoundedSurface(surface.as<typename IfcEntityTypesT::IfcBoundedSurface(),
 							pos);
 					}
 
 					// (2/3) IfcElementarySurface SUBTYPE of IfcSurface
-					if (surface.isOfType<typename IfcEntityTypesT::IfcElementarySurface>()) const throw(...) {
-						return std::shared_ptr<carve::input::PolylineSetData> polylineData = convertIfcElementarySurface(surface.as<typename IfcEntityTypesT::IfcElementarySurface(),
+					if (surface.isOfType<typename IfcEntityTypesT::IfcElementarySurface>()) const throw(...) 

```

Comment:
```suggestion
					if (surface.isOfType<typename IfcEntityTypesT::IfcElementarySurface>()) 
```
No `const throw(...)` after an `if` clause. These are specifiers for function definitions.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/151/commits/8f06b96decdf1044638e00fb33a084d2485b8e94